src/lib/pages/frontsite/PatronProfile/PatronCurrentDocumentRequests/PatronCurrentDocumentRequests.js

```diff
@@ -207,6 +207,12 @@ class PatronCurrentDocumentRequests extends Component {
     { title: 'Created', field: 'created', formatter: dateFormatter },
     { title: 'Authors', field: 'metadata.authors' },
     { title: 'Publication year', field: 'metadata.publication_year' },
+    {
+      title: 'Request type',
```

Comment:
I noticed in the screenshot that the request type appears in capital `BUY`, in other cases we introduce labels so we have full control on what to display per field.

Reply:
I agree. Also `Purchase` sounds better than `Buy`